gm4_ambrosia/data/gm4/advancements/ambrosia_bottled_honey.json

```diff
@@ -0,0 +1,19 @@
+{
+  "display":{
+    "icon":{
+      "item":"potion",
+      "nbt":"{display:{Name:\"\\\"ambrosia logo\\\"\"},CustomPotionColor:16754944}"
```

Comment:
What'd be the correct `CustomModelData`?

Reply:
Since this is just an advancement it can easily be patched after release. We'll release the module with the custom name and are happy to patch this once people have settled on a value for `CustomModelData`.